frontend/src/bundles/video_editor/components/timeline/subcomponents/timeline-view.tsx

```diff
@@ -0,0 +1,101 @@
+import { format, hoursToMilliseconds, minutesToMilliseconds, secondsToMilliseconds } from 'date-fns';
+import  { type ItemDefinition, type RowDefinition } from 'dnd-timeline';
+import { groupItemsToSubrows, useTimelineContext } from 'dnd-timeline';
+import { useMemo } from 'react';
+
+import { Item } from './item.js';
+import { Row } from './row.js';
+import { Subrow } from './subrow.js';
+import { type MarkerDefinition } from './timeaxis.js';
+import { TimeAxis } from './timeaxis.js';
+import { TimeCursor } from './timecursor.js';
+
+const timeAxisMarkers: MarkerDefinition[] = [
```

Comment:
It is better to create a separate file with this constant.

Reply:
Should I create the file in /helpers?